app/views/users/show/_timeline.html.erb

```diff
@@ -3,40 +3,45 @@
     <h3 class="title is-3">Your contributions</h3><br/>
     <table class="is-narrow is-fullwidth">
       <tbody>
-        <% scoring_pull_requests.map do |pr| %>
-          <%= render partial: 'users/show/pr/pr', locals: { pr: pr } %>
-        <% end %>
-
-        <% if show_timer %>
+        <% @valid_pr = 0 %>
+        <% if show_congratulations %>
           <tr>
             <td colspan="2">
               <div class="winning-msg-cont">
-                <h2 class="winning-title">You're almost there!</h2>
-                <span class="winning-desc">🎉 You’ve submitted the four required PRs for the Hacktoberfest challenge! So long as your PRs successfully pass the review period, they’ll become valid — which means you’ll have officially completed this year’s challenge!</span>
+                <h2 class="winning-title">You did it! 🎉</h2>
+                <span class="winning-desc">You've made four eligible PRs for Hacktoberfest — which means you officially completed this year’s challenge!</span>
               </div>
             </td>
           </tr>
         <% end %>
 
-        <% if show_congratulations %>
+        <% scoring_pull_requests.map.with_index do |pr| %>
+          <% if pr.eligible? || pr.waiting? %>
+            <%= render partial: 'users/show/pr/pr', locals: { pr: pr, valid_pr: @valid_pr +=1, show_congratulations: show_congratulations, pr_type: 1 }  %>
+          <% else %>
+            <%= render partial: 'users/show/pr/pr', locals: { pr: pr, valid_pr: @valid_pr, show_congratulations: show_congratulations, pr_type: 0 }  %>
+          <% end %>
+        <% end %>
+
+        <% if scoring_pull_requests.length == 4 && !show_congratulations %>
```

Comment:
Why are we using this logic, and not just the original `show_timer`? That was based on user state, which ensures we're showing the message when expected, whereas this adds more complex logic to achieve hopefully the same thing?